diff --git a/node_modules/@expo/cli/build/src/start/server/metro/externals.js b/node_modules/@expo/cli/build/src/start/server/metro/externals.js
index 8341467..67270dc 100644
--- a/node_modules/@expo/cli/build/src/start/server/metro/externals.js
+++ b/node_modules/@expo/cli/build/src/start/server/metro/externals.js
@@ -79,7 +79,7 @@ function tapNodeShimContents(moduleId) {
 async function tapNodeShims(projectRoot) {
     const externals = {};
     for (const moduleId of NODE_STDLIB_MODULES){
-        const shimDir = _path.default.join(projectRoot, METRO_EXTERNALS_FOLDER, moduleId);
+        const shimDir = _path.default.join(projectRoot, METRO_EXTERNALS_FOLDER, moduleId.replace(/:/g, '-'));
         const shimPath = _path.default.join(shimDir, "index.js");
         externals[moduleId] = shimPath;
         if (!_fs.default.existsSync(shimPath)) {
